src/components/app/details/triggerView/EmptyStateCIMaterial.tsx

```diff
@@ -60,12 +62,21 @@ export class EmptyStateCIMaterial extends Component<EmptyStateCIMaterialProps> {
       </EmptyState>
     }
     else {
-      return <EmptyState >
-        <EmptyState.Image>{img}</EmptyState.Image>
-        <EmptyState.Title>{title}</EmptyState.Title>
-        <EmptyState.Subtitle>{subtitle}</EmptyState.Subtitle>
-        <EmptyState.Button>{cta}</EmptyState.Button>
-      </EmptyState>
+      return (
+          <EmptyState>
+              <EmptyState.Image>{img}</EmptyState.Image>
+              <EmptyState.Title>{title}</EmptyState.Title>
+              <EmptyState.Subtitle className="mb-0">{subtitle}</EmptyState.Subtitle>
+              <EmptyState.Button>{cta}</EmptyState.Button>
+              <EmptyState.Subtitle>
+                  {this.props.isWebHook ? (
+                      <span className="learn-more__href cursor" onClick={this.props.toggleWebHookModal}>
+                          View all incoming webhook payloads
+                      </span>
+                  ) : null}
```

Comment:
set `subtitle` instead of null as it was done before.

Reply:
As per discussion changes are done